tools/jsopcode.py

```diff
@@ -1,7 +1,9 @@
-#!/usr/bin/env python -B
+#!/usr/bin/env python3 -B
```

Comment:
Doesn't this require `-S` too?

Reply:
Oh! This is imported directly from our tree. Ok.

Reply:
It has some modifications to output GFM markdown, but otherwise yes.